tests/test_views.py

```diff
@@ -358,15 +357,16 @@ def test_not_set_subset_X_dask(matrix_type_no_gpu, subset_func):
     with pytest.warns(ad.ImplicitModificationWarning, match=r".*X.*"):
         subset.X[:, internal_idx] = 1
     assert not subset.is_view
-    assert not np.any(asarray(adata.X != orig_X_val))
+    assert not np.any(to_dense(adata.X != orig_X_val))
 
     assert init_hash == tokenize(adata)
     assert isinstance(subset.X, type(adata.X))
 
 
 @IGNORE_SPARSE_EFFICIENCY_WARNING
-def test_set_scalar_subset_X(matrix_type, subset_func):
-    adata = ad.AnnData(matrix_type(np.zeros((10, 10))))
+@pytest.mark.array_type(skip={Flags.Disk, *CUPY_SPARSE, *DASK_SPARRAY})
```

Comment:
I think this sort of thing will now run these tests on gpu whereas they did not before?